src/Aspire.Hosting/Dashboard/ResourceCollection.cs

```diff
@@ -8,26 +8,16 @@
 namespace Aspire.Hosting.Dashboard;
 
 /// <summary>
-/// Builds a collection of resources by integrating incoming changes from a channel,
-/// and allowing multiple subscribers to receive the current resource snapshot and future
-/// updates.
+/// Builds a collection of resources by integrating incoming resource changes,
+/// and allowing multiple subscribers to receive the current resource collection
+/// snapshot and future updates.
 /// </summary>
-internal sealed class ResourceCollection
+internal sealed class ResourceCollection(CancellationToken cancellationToken)
```

Comment:
I'd like a new name for this. There are two types in hosting called ResourceCollection, and this one isn't a collection.